icu4c/source/test/intltest/dtifmtts.cpp

```diff
@@ -1673,7 +1673,9 @@ void DateIntervalFormatTest::testFormattedDateInterval() {
         const char16_t* message = u"FormattedDateInterval test 1";
         const char16_t* expectedString = u"July 20 \u2013 25, 2018";
         LocalPointer<Calendar> input1(Calendar::createInstance("en-GB", status));
+	if (status.errIfFailureAndReset()) { return; }
```

Comment:
FYI ok, but only the second one is really necessary. Line 1677 won't do anything if failure, we just need to catch the failure before dereferencing the pointers on lines 1679 & 1680.

Reply:
Ack.

Reply:
All indents corrected if needed.